controllers/podnetworkchaos/netutils/cidr.go

```diff
@@ -43,29 +48,50 @@ func ResolveCidrs(names []string) ([]string, error) {
 }
 
 // ResolveCidr converts cidr/ip/domain into cidr
-func ResolveCidr(name string) ([]string, error) {
-	_, ipnet, err := net.ParseCIDR(name)
+func ResolveCidr(name string) ([]v1alpha1.CidrAndPort, error) {
+	var toResolve string
+	var port uint16
+
+	if host, portStr, err := net.SplitHostPort(name); err == nil {
+		toResolve = host
+		port64, err := strconv.ParseUint(portStr, 10, 16)
+		if err != nil {
+			return nil, fmt.Errorf("failed to parse port %w", err)
```

Comment:
`errors.Errorf("parse port %w", err) ` , You can refer to https://github.com/chaos-mesh/rfcs/blob/main/text/2021-09-27-refine-error-handling.md.

Reply:
Done!